### DISCUSSION TITLE
Bravo + Y a t'il une initiative similaire pour GAZPAR ?

### DISCUSSION THREAD

**Discussion by dgillet2** [bokub/conso-api#15] [Idées]
Bonjour,
Je cherchais depuis longtemps une solution à ce cauchemar de captcha.
C'est un comble de devoir passer par un tiers qui contractualise et paye pour m'envoyer mes propres données.

En tout cas bravo pour cette solution : elle fonctionne parfaitement et tu l'as abondamment documentée.
As-tu connaissance d'une initiative similaire pour GAZPAR ?
J'ai bien vu [gazpar2mqtt](https://github.com/yukulehe/gazpar2mqtt#gazpar2mqtt), mais obsolète depuis la mise en place du captcha justement.

Merci pour ce dev en tout cas

**Comment by bokub** [comment]
Non désolé, je n'utilise pas de gaz naturel et je ne me suis donc jamais penché sur la question
